Formula/opencv.rb

```diff
@@ -91,10 +92,12 @@ def install
 
     mkdir "build" do
       system "cmake", "..", *args
+      inreplace "modules/core/version_string.inc", "#{HOMEBREW_SHIMS_PATH}/mac/super/", ""
```

Comment:
Does this still use shims at build-time?

Reply:
The part of this file being patched is only part of a build info screen, and not any parsed variables used in the build process.